adapters/InMobi/InMobiAdapter/GADMAdapterInMobiUtils.m

```diff
@@ -114,25 +125,32 @@ void GADMAdapterInMobiSetTargetingFromExtras(GADInMobiExtras *_Nullable extras)
 }
 
```

Comment:
@kshtgarg21 Actually we can add the below (or similar) in this class:
```
void GADMAdapterInMobiLog(NSString *_Nonnull format, ...) {
  va_list arguments;
  va_start(arguments, format);
  NSString *log = [[NSString alloc] initWithFormat:format arguments:arguments];
  va_end(arguments);

  NSLog(@"GADMediationAdapterInMobi - %@", log);
}
```


Then in the other places we can use it for logging.

Reply:
done